Kconfig

```diff
@@ -7,6 +7,8 @@ choice WIREGUARD_ESP_ADAPTER_SELECTION
         bool "TCP/IP Adapter (pre esp-idf v4.1, ESP8266 RTOS SDK)"
     config WIREGUARD_ESP_NETIF
         bool "ESP-NETIF"
+    config WIREGUARD_ESP_NETIF_ETHERNET
```

Comment:
The above config param could be renamed as `WIREGUARD_ESP_NETIF_WIFI` so as to avoid confusion

Reply:
Renamed to WIREGUARD_ESP_NETIF_WIFI